.github/workflows/ci.yml

```diff
@@ -76,8 +76,23 @@ jobs:
     outputs:
       os: ${{ steps.setup-os-matrix.outputs.os }}
 
-  build-and-test:
-    needs: setup-os-matrix
+  build:
+    needs: check-style
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup the latest .NET 5 SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+
+      - name: Build
+        run: dotnet build --configuration=Release -p:TreatWarningsAsErrors=true Src
+
+  test:
+    needs: [setup-os-matrix, build]
```

Comment:
`test` should run in parallel to `build` as it does not depend on any of its output - it will speed up the global CI time

Reply:
```suggestion
    needs: [setup-os-matrix]
```